external/fold.js

```diff
@@ -381,6 +381,8 @@ convert.edges_vertices_to_edges_faces_edges = function(fold) {
     ];
   }
   ref1 = fold.faces_vertices;
+  // NOT PART OF ORIGINAL LIBRARY
+  fold.faces_edges = [];
```

Comment:
What's the purpose of this line?

Reply:
There's a bug in the FOLD API where this method is supposed to create the `faces_edges` attribute if it doesn't exist, but never initializes it like it does in other methods, which leads it to try to assign values to something that doesn't exist. This line initializes that.

Reply:
Ahhh, that makes sense. Thank you!